packages/DataTable/stories/ShowHideColumns.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import DataTable, { RowHeight, Sort } from "../src";
+import fixtures from "./fixtures";
+import { viewPortHeight } from "./helpers";
+import moment from "moment";
+import ColumnManaging from "../src/components/Navigation/components/ColumnManaging";
+
+const mockData = fixtures(1);
+function App() {
+  return (
+    <DataTable keygen="id" data={mockData} height={viewPortHeight()}>
+      <DataTable.Navigation>
+        <Sort />
+        <RowHeight />
+        <ColumnManaging />
```

Comment:
Maybe later we can asked for better name than RowHeight and ColumnManaging, but for now is cool :)